boards.txt

```diff
@@ -6778,6 +6778,189 @@ adafruit_feather_esp32_v2.menu.DebugLevel.verbose=Verbose
 adafruit_feather_esp32_v2.menu.DebugLevel.verbose.build.code_debug=5
 
 
+##############################################################
+
+adafruit_feather_esp32s3.name=Adafruit Feather ESP32-S3 No PSRAM
+adafruit_feather_esp32s3.vid.0=0x239A
+adafruit_feather_esp32s3.pid.0=0x8113
+adafruit_feather_esp32s3.vid.1=0x239A
+adafruit_feather_esp32s3.pid.1=0x0113
+adafruit_feather_esp32s3.vid.1=0x239A
+adafruit_feather_esp32s3.pid.1=0x8114
+
+adafruit_feather_esp32s3.upload.tool=esptool_py
+adafruit_feather_esp32s3.upload.maximum_size=1310720
+adafruit_feather_esp32s3.upload.maximum_data_size=327680
+adafruit_feather_esp32s3.upload.flags=
+adafruit_feather_esp32s3.upload.extra_flags=0x410000 "{runtime.platform.path}/variants/{build.variant}/tinyuf2.bin"
+adafruit_feather_esp32s3.upload.use_1200bps_touch=true
+adafruit_feather_esp32s3.upload.wait_for_upload_port=true
+
+adafruit_feather_esp32s3.serial.disableDTR=false
+adafruit_feather_esp32s3.serial.disableRTS=false
+
+adafruit_feather_esp32s3.build.tarch=xtensa
+adafruit_feather_esp32s3.build.bootloader_addr=0x0
+adafruit_feather_esp32s3.build.target=esp32s3
+adafruit_feather_esp32s3.build.mcu=esp32s3
+adafruit_feather_esp32s3.build.core=esp32
+adafruit_feather_esp32s3.build.variant=adafruit_feather_esp32s3_nopsram
+adafruit_feather_esp32s3.build.board=ADAFRUIT_FEATHER_ESP32S3_NOPSRAM
+
+adafruit_feather_esp32s3.build.usb_mode=1
+adafruit_feather_esp32s3.build.cdc_on_boot=1
+adafruit_feather_esp32s3.build.msc_on_boot=0
+adafruit_feather_esp32s3.build.dfu_on_boot=0
+adafruit_feather_esp32s3.build.f_cpu=240000000L
+adafruit_feather_esp32s3.build.flash_size=8MB
+adafruit_feather_esp32s3.build.flash_freq=80m
+adafruit_feather_esp32s3.build.flash_mode=dio
+adafruit_feather_esp32s3.build.boot=qio
+adafruit_feather_esp32s3.build.partitions=default
+adafruit_feather_esp32s3.build.defines=
+adafruit_feather_esp32s3.build.loop_core=-DARDUINO_RUNNING_CORE=1
+adafruit_feather_esp32s3.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+
+adafruit_feather_esp32s3.menu.USBMode.default=USB-OTG
+adafruit_feather_esp32s3.menu.USBMode.default.build.usb_mode=0
+adafruit_feather_esp32s3.menu.USBMode.default.upload.use_1200bps_touch=true
+adafruit_feather_esp32s3.menu.USBMode.default.upload.wait_for_upload_port=true
+adafruit_feather_esp32s3.menu.USBMode.hwcdc=Hardware CDC and JTAG
+adafruit_feather_esp32s3.menu.USBMode.hwcdc.build.usb_mode=1
+adafruit_feather_esp32s3.menu.USBMode.hwcdc.upload.use_1200bps_touch=false
+adafruit_feather_esp32s3.menu.USBMode.hwcdc.upload.wait_for_upload_port=false
+
+adafruit_feather_esp32s3.menu.CDCOnBoot.cdc=Enabled
+adafruit_feather_esp32s3.menu.CDCOnBoot.cdc.build.cdc_on_boot=1
+adafruit_feather_esp32s3.menu.CDCOnBoot.default=Disabled
+adafruit_feather_esp32s3.menu.CDCOnBoot.default.build.cdc_on_boot=0
```

Comment:
ok i have had no issues flashing thru CDC with tinyusb other than reset needs to be pressed.

Reply:
That is because you have tried just recently :D a week ago even that was not working. We will try to figure it out, but for now that limitation stands and does not exist with the built-in cdc. I guess you want to make sure that PID/VID are yours? You do know that Download mode changes them back to ours, right?